src/backend/marsha/core/serializers/video.py

```diff
@@ -310,6 +310,75 @@ class InitLiveStateSerializer(serializers.Serializer):
     type = serializers.ChoiceField(LIVE_TYPE_CHOICES)
 
 
+class LiveRegistrationSerializer(serializers.ModelSerializer):
+    """Serializer for liveRegistration model."""
+
+    class Meta:  # noqa
+        model = LiveRegistration
+        fields = (
+            "email",
+            "id",
+            "lti_consumer_site",
+            "lti_user_id",
+            "should_send_reminders",
+            "video",
+        )
+        read_only_fields = (
+            "lti_consumer_site",
+            "lti_user_id",
+            "video",
+        )
+
+    # Make sure video UUID is converted to a string during serialization
+    video = serializers.PrimaryKeyRelatedField(
+        read_only=True, pk_field=serializers.CharField()
+    )
+
+    def create(self, validated_data):
+        """Force the video field to the video of the JWT Token if any.
+
+        Parameters
+        ----------
+        validated_data : dictionary
+            Dictionary of the deserialized values of each field after validation.
+
+        Returns
+        -------
+        dictionary
+            The "validated_data" dictionary is returned after modification.
+
+        """
+        # user here is a video as it comes from the JWT
+        # It is named "user" by convention in the `rest_framework_simplejwt` dependency we use.
+        user = self.context["request"].user
+        video = Video.objects.get(pk=user.id)
+
+        if video.is_scheduled is not True:
```

Comment:
```suggestion
        if video.is_scheduled is False:
```

Reply:
None probably equal to False, I should have tested it